barretenberg/cpp/CMakePresets.json

```diff
@@ -422,6 +412,29 @@
         "MULTITHREADING": "ON"
       }
     },
+    {
+      "name": "wasm-threads-dbg",
+      "displayName": "Build for debug WASM",
+      "binaryDir": "build-wasm-threads-dbg",
+      "description": "Build with wasi-sdk to create debug wasm",
+      "inherits": "wasm",
+      "environment": {
+        "CMAKE_BUILD_TYPE": "Debug"
+      },
+      "cacheVariables": {
+        "MULTITHREADING": "ON"
+      }
+    },
+    {
+      "name": "wasm-threads-assert",
```

Comment:
Added asserts build for wasm, _much_ faster to build than dbg. Considered making it the default but idk, might affect performance if we use heavy asserts.